BringgSDK.js

```diff
@@ -815,6 +815,25 @@ var BringgSDK = (function () {
     }
   };
 
+
+  //==============================================
+  // Lazy Masked Phone Number
+  //==============================================
+  module.getDriverPhone = function (uuid, cb) {
+    if (!uuid) {
+      cb({status: 'error', message: 'No shared_uuid provided'});
```

Comment:
on the watch apis i see we also provide {rc: module.RETURN_CODES.missing_params, ...} in the callback.
this is nice (although we don't do it for other apis like notes and such sop you decide)